pkg/api/validation/validation.go

```diff
@@ -103,7 +112,7 @@ func nameIsDNSSubdomain(name string, prefix bool) (bool, string) {
 	if util.IsDNSSubdomain(name) {
 		return true, ""
 	}
-	return false, "name must be lowercase letters and numbers, with inline dashes or periods"
+	return false, "Name " + name + DnsSubdomainErrorMsg
```

Comment:
should not need "Name " or name here - that should be created as part of the ValidationError.Error() process.  Just return DnsSubdomainErrorMsg